frontend/components/customizer/customizer.tsx

```diff
@@ -11,6 +12,57 @@ import {
   CopiedContentIndicatorHandle,
 } from './copied-content-indicator'
 
+
+function generateBuiltBadgeUrl({
+  baseUrl,
+  queryString,
+  path,
+}: {
+  baseUrl: string,
+  queryString: string | undefined,
+  path: string,
+}): string {
+  const suffix = queryString ? `?${queryString}` : ''
+  return `${baseUrl}${path}${suffix}`
+}
+
+function renderLivePreview({
```

Comment:
In general, prefer React components to helper functions where possible. React components need to start with a capital letter, and can then be used like this: `<LivePreview baseUrl={...} ... />`

```suggestion
function LivePreview({
```